lib/if.h

```diff
@@ -222,6 +222,7 @@ struct interface {
 	 */
 	ifindex_t ifindex;
 #define IFINDEX_INTERNAL	0
+#define IFINDEX_UNKNOWN		IFINDEX_INTERNAL
```

Comment:
I don't think we need a separate `#define` for this, I would just pass `0` directly. It's only used inside `zebra/if_netlink.c` so IMHO it should be clear by looking at the code what the `0` means.